packages/sprotty/tsconfig.json

```diff
@@ -8,5 +8,7 @@
   ],
   "exclude": [
     "**/*.spec.ts", "**/*.spec.tsx", "src/utils/test-helper.ts"
-  ]
+  ],
+  "experimentalDecorators": true,
+  "emitDecoratorMetadata": true
```

Comment:
Shouldn't these be part of the `compilerOptions`? The base tsconfig should already include them.

Reply:
ups yeah, i think forgot to remove them again. I'll do that right away